vendor/acrios/acr-cv-101l-m-d.yaml

```diff
@@ -1,38 +1,90 @@
-name: M-Bus IoT converter
-description: The Acrios M-Bus IoT converter is an M-Bus to LoRaWAN® converter that can be used for reading electricity meters and calorimeters. The M-Bus converter supports connections of up to 5 UL (Unit Loads) and communication is based on the EN 137 57-3 protocol.
+name: M-Bus to LoRaWAN converter
+description: M-Bus to LoRaWAN converter is used to retrofit ANY M-Bus meter with LoRaWAN communication
+
 hardwareVersions:
-  - version: 'v3.7'
+  - version: 'ACR-CV-101L-M-DEAC_v1.1'
     numeric: 1
+
 firmwareVersions:
   - # Firmware version
-    version: 'e8d17c15795f'
```

Comment:
Please do not change the existing values of `hardwareVersions` and `firmwareVersions`.

Reply:
Fixing it right now

Reply:
Have you ran `make validate fmt` already as well? If not do so please.

Reply:
Before commit I ran `make validate`

Reply:
Could you run make validate fmt too? That will take care of formatting as well.

Reply:
Apologies for the oversight earlier. I was multitasking and didn't notice that I missed running `make validate fmt` and the FW/HW versions modifying. I'll take care of it now.

Reply:
No worries!

Reply:
I ran `make validate fwt` and output was:

```
make: *** No rule to make target 'fwt'.  Stop.
```

Reply:
Done, git shows no changes

Reply:
Alright thank you. My colleague @Jaime-Trinidad will do the final check and approval.